app/views/comments/_comment.html.slim

```diff
@@ -10,7 +10,7 @@
       h5 #{comment.title}
       p #{comment.content}
       p.reply.comment-button
-        - if comment.can_delete?(current_user) && !current_user.banned?
+        - if comment.can_delete?(current_user) && active?
```

Comment:
probably #active? should be moved to #can_delete?
or inactive user can delete to?